examples/column_transformer_mixed_types.py

```diff
@@ -0,0 +1,101 @@
+"""
+===================================
+Column Transformer with Mixed Types
+===================================
+
+This example demonstrates how to use
+:class:`sklearn.compose.ColumnTransformer` on a dataset consisting of
+hereogeneous data types. A subset of numeric and categorical features
+are selected and :class:`sklearn.compose.ColumnTransformer` is used to
+apply two non-sequential preprocessing pipelines, one for each data
+type:
+
+* Numeric data: missing value imputation and standard scaling
+* Categorical data: missing value imputation and one-hot encoding.
```

Comment:
Maybe say for categorical that missing values are replaced with a new category and for numeric data with the mean? (that's the default, right?)